zync.py

```diff
@@ -28,11 +28,15 @@ class ZyncConnectionError(Exception):
 class ZyncPreflightError(Exception):
     pass
 
-current_dir = os.path.dirname(os.path.abspath(__file__)) 
-config_path = os.path.join(current_dir, 'config.py')
-if not os.path.exists(config_path):
-  raise ZyncError('Could not locate config.py, please create.')
-from config import *
+if os.environ.get('ZYNC_URL'):
```

Comment:
Hi Sam,

I'm working right now on a PR that allows passing zync_url to the constructor of zync.Zync. Would that work for you? My PR should go live next week.